src/com/jester/controller/Controller.java

```diff
@@ -50,7 +50,7 @@ public Music SetNewMusic(Music insert_music) {
     }
     
     public String[] GetAllStoredAlbums() {
-        String[] albums = new String[5];
+        String[] albums = new String[1000];

```

Comment:
Updated the size to prevent issues with scaling

Reply:
This one is for making the return more viable.